src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ManualPaymentSubmittedHandler.java

```diff
@@ -50,7 +51,7 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
             log.info("Sending letter correspondence to applicant for case: {}", caseDetails.getId());
             CaseDocument caseDocument =
                 service.generateManualPaymentLetter(caseDetails, userAuthorisation, APPLICANT);
-            printService.sendDocumentForPrint(caseDocument, caseDetails);
+            printService.sendDocumentForPrint(caseDocument, caseDetails, CCDConfigConstant.APPLICANT);
```

Comment:
Can be use the static import of APPLICANT

Reply:
can't do- as one already with same in static import